app/Repositories/BatchRepository.php

```diff
@@ -10,4 +10,16 @@ public function model()
     {
         return \App\Models\Batch::class;
     }
+
+    public function listBatchesArray()
+    {
+        $array = [];
+        $results = $this->model->all();
+        foreach ($results as $result) {
+            $array[$result->id] = 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/BatchRepository.php</br>**Line**: 19</br>Whitespace found at end of line